oap-server/server-configuration/configuration-api/src/main/java/org/apache/skywalking/oap/server/configuration/api/ConfigWatcherRegister.java

```diff
@@ -110,6 +110,67 @@ void configSync() {
 
             LOGGER.trace("Current configurations after the sync." + LINE_SEPARATOR + register.toString());
         });
+
+        GroupConfigsSync(configTable);
+    }
+
+    private void GroupConfigsSync(Optional<ConfigTable> configTable) {
+        configTable.ifPresent(config -> {
```

Comment:
Try to reduce the nested levels

```suggestion
        configTable.ifPresent(this::groupConfigsSync);
    }

    private void groupConfigsSync(ConfigTable config) {
```

Also the first letter should be lowercase